pixl_dcmd/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include src/pixl_dcmd/resources/*.yml
```

Comment:
I thin you can do this in pyproject.toml which IIRC we've done in pytest-pixl using package-data

Reply:
Indeed you can, I thought I'd tried it that way but I must have mucked up the syntax.